src/CommunityToolkit.Maui.UnitTests/Converters/IntToBoolConverter_Tests.cs

```diff
@@ -25,16 +27,17 @@ public void IndexToArrayConverterBack(bool value, int expectedResult)
 	{
 		var intToBoolConverter = new IntToBoolConverter();
 
-		var result = (int)intToBoolConverter.ConvertBack(value, typeof(bool), null, CultureInfo.CurrentCulture);
+		var result = intToBoolConverter.ConvertBack(value, typeof(bool), null, CultureInfo.CurrentCulture);
+		var typedResult = intToBoolConverter.ConvertBackTo(value);
 
 		Assert.Equal(result, expectedResult);
+		Assert.Equal(typedResult, expectedResult);
 	}
 
 	[Theory]
 	[InlineData(2.5)]
 	[InlineData("")]
-	[InlineData(null)]
-	public void InValidConverterValuesThrowArgumenException(object value)
+	public void InValidConverterValuesThrowArgumentException(object value)
```

Comment:
Let's fix this small typo from a previous PR

```suggestion
	public void InvalidConverterValuesThrowArgumentException(object value)
```